src/sentry/search/events/datasets/metrics.py

```diff
@@ -174,6 +174,42 @@ def function_converter(self) -> Mapping[str, fields.MetricsFunction]:
                     result_type_fn=self.reflective_result_type(),
                     default_result_type="duration",
                 ),
+                fields.MetricsFunction(
+                    "count_if",
+                    required_args=[
+                        fields.MetricArg(
+                            "if_col",
+                            allowed_columns=["release"],
+                        ),
+                        fields.SnQLStringArg(
+                            "if_val", unquote=True, unescape_quotes=True, optional_unquote=True
```

Comment:
Please update this to match the syntax of the other datasets, and include the condition argument as well?